proto/src/determined/api/v1/api.proto

```diff
@@ -142,6 +142,11 @@ service Determined {
         option (google.api.http) = {get: "/api/v1/experiments"};
         option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {tags: "Experiments"};
     }
+    // Get a list of experiment labels
```

Comment:
nit:
- period at the end
- do you think it'll be helpful to say these are gonna be a list of _unique labels_ and across all experiments. also, would we wanna mention the default sorting anywhere?

Reply:
Fixed